apis/postgresql/v1alpha1/postgresql_standalone_operator_config_types.go

```diff
@@ -66,6 +69,12 @@ type PostgresqlStandaloneOperatorConfig struct {
 	Status PostgresqlStandaloneConfigStatus       `json:"status,omitempty"`
 }
 
+// Persistence contains default PVC settings.
+type Persistence struct {
+	// storageClassName is the name of the StorageClass required by the claim.
+	StorageClassName *string `json:"storageClassName,omitempty" protobuf:"bytes,5,opt,name=storageClassName"`
```

Comment:
I got the value from the Kubernetes PersistentVolumeClaim definition. When a default storage class is being used I apply it as nil pointer to the StorageClass instead of "" which generates an error

Reply:
ah :+1: